src/StockportWebapp/Views/stockportgov/Article/Article2024.cshtml

```diff
@@ -8,61 +8,64 @@
 }
 
 <div class="grid-container-full-width" style="background-color:white;">
+    <partial name="Breadcrumb2023" model="Model.Article.Breadcrumbs" />
+
     <div class="grid-container">
         <div class="grid-100">
-            <div class="grid-70">
-                <div class="placeholder placeholder-bg-teal placeholder__left-align placeholder__padded-top">
-                    <div class="grid-container">
-                        <p>Breadcrumbs</p>
-                    </div>
-                </div>
-
-                <div class="placeholder placeholder-bg-green placeholder__left-align placeholder__padded-top placeholder__padded-bottom" id="article-page-content">
-                    <div class="grid-container">
-                        <h1 class="placeholder-article-title">Title</h1>
-                    </div>
-                </div>
+            <div class="article--70">
+                <h1 class="directory__title">@Model.Article.Title</h1>
             </div>
 
-            <div class="grid-70">
-                <div class="placeholder placeholder-bg-grey placeholder-small">
-                    <h2 class="placeholder-article-header-description">Image</h2>
-                </div>
+            <section class="article">
+                <div class="article--70">
+                    @if(!string.IsNullOrEmpty(Model.Article.Image))
```

Comment:
Could you add a space between the if and the opening parenthesis please (won't add a comment for all of them but I saw a few)